frontend/src/pages/order-status/libs/components/buttons-section/buttons-section.tsx

```diff
@@ -1,49 +1,55 @@
 import { Button } from '~/libs/components/components.js';
 
+import { OrderStatus } from '../../enums/order-status.enum.js';
```

Comment:
Since the file has been created, we can fix it here.
```suggestion
import { OrderStatus } from '../../enums/enums.js';
```